### PR TITLE
PSQLADM-415 Update contributing.md

### PR DESCRIPTION
**Pull request by patrickbirch**
A  contributing.md
A  docs/_images/add-jira.png
A  docs/_images/create-issue.png
A  docs/_images/jira-ticket.png
A  docs/_images/new-topic.png

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+## Add a forum topic
+
+In the [Percona Product Documentation category](https://forums.percona.com/c/percona-product-documentation/71) in the Percona Community Forum, select **New Topic**. Complete the form and select **Create Topic** to add the topic to the forum.
+
+<img src= "./docs/_images/new-topic.png" alt ="create a topic" width = 40% height = 40% />
+
+## Request a change with a Jira issue
+
+If you would rather not [contribute to the documentation yourself](#contribute-to-documentation-yourself) let us know about the issue by adding a Jira ticket. Use the following procedure to create a Jira ticket:
```
Comment (alina-derkach-oaza) [inline contributing.md:27]: Add comma after [contribute to the documentation yourself](#contribute-to-documentation-yourself)

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+In the Contact Us section, select the link in "open a JIRA ticket" 
+
+<img src = "./docs/_images/jira-ticket.png" alt="JIRA ticket" width = 30% height = 30% />
+
+- Select the **open a JIRA ticket** link in the Contact Us section on the bottom of the page. This action opens the [Jira issue tracker](https://jira.percona.com/projects/PSQLADM/issues) for the doc project.
```
Comment (alina-derkach-oaza) [inline contributing.md:33]: on the bottom of the page -> at the bottom of the page

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Using either the [Edit documentation online with GitHub](#edit-documentation-online-with-github) method or the [Edit documentation locally](#edit-documentation-locally) method to make changes to the documentation and create a pull request. 
+ 
+
+### What you should know
+
+Most of the document is in plain text, but you may use [Markdown] (https://www.markdownguide.org/) to add syntax elements (notes, tables, and so on) to the documentation. 
+
+### What happens after you create the pull request
+
+A core developer reviews your request and either comments or approves the request. A core developer merges the approved request to the **main** branch usually when a newer version of the product is released.
```
Comment (alina-derkach-oaza) [inline contributing.md:59]: A core developer - should we say a technical writer here?

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+In the upper right corner, select the pencil icon to open the source file in the GitHub editor. In this editor, you can make the changes, view the changes in the Preview tab, and create a pull request. The source `.md` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page using the Markdown syntax.
+
+You should review your changes on the **Preview** tab.
+
+Commit your changes to a pull request.
+
+In the **Commit changes** section, add a summary (72 characters or less) of what was changed
+
```
Comment (alina-derkach-oaza) [inline contributing.md:76]: Add a full stop

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+Commit your changes to a pull request.
+
+In the **Commit changes** section, add a summary (72 characters or less) of what was changed
+  
+Then select the **Create a new branch for this commit and start a pull request** option. Accept the name for the branch and then Select **Commit changes**.
+
+GitHub creates a branch and a commit for your changes. The operation loads a page that shows the following:
+
+-  a base branch where you offer your changes
```
Comment (alina-derkach-oaza) [inline contributing.md:81]: -  a - remove extra space

File: contributing.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+    ```shell
+        git checkout -b <my_changes>
+    ```
+
+6. The structure of the repository is:
+
+    <img src= "./docs/_images/treeview.png" alt="tree view of directory" width = 20% height = 20%)/>
+
+7. Work in the `/docs` directory. Add code examples, if necessary. We recommend that you check your changes using either a Preview built into your editor (if you have one) or build HTML to check it there. You can build HTML [on your machine](#Installing-Sphinx-and-building-locally) or [using Docker](#Using-Docker).
```
Comment (alina-derkach-oaza) [inline contributing.md:128]: on your machine](#Installing-Sphinx-and-building-locally) -> we do not use Sphinx any more

File: contributing.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+GitHub creates a branch and a commit for your changes. The operation loads a page that shows the following:
+
+-  a base branch where you offer your changes
+  
+- your commit message
+  
+- a visual representation of your changes against the original page 
+
+You can make a review the page and click **Create pull request**.
```
Comment (alina-derkach-oaza) [inline contributing.md:87]: You can make a review the page and click **Create pull request**.  -->. You can review the page and click **Create pull request**.

File: contributing.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+!!! note
+
+    We appreciate your work but the pull request may be redone to meet internal requirements.
+
+### Edit documentation online with GitHub
+
+In the upper right corner, select the pencil icon to open the source file in the GitHub editor. In this editor, you can make the changes, view the changes in the Preview tab, and create a pull request. The source `.md` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page using the Markdown syntax.
```
Comment (alina-derkach-oaza) [inline contributing.md:69]: Should we add here a link where the user can check Markdown syntax?

File: contributing.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+        cd <directory name>/proxysql-admin-tool-doc
+    ```
+
+3. Add the remote origin repository:
+
+    ```shell
+        git remote add origin https://github.com/percona/proxysql-admin-tool-doc.git
+    ```
+
+4. Checkout the appropriate branch and pull the latest changes from origin
```
Comment (alina-derkach-oaza) [inline contributing.md:112]: add a colon

File: contributing.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+
+    ```shell
+        git checkout main && git pull origin main
+    ```
+
+5. Create a separate branch for your changes
```
Comment (alina-derkach-oaza) [inline contributing.md:118]: add a colon

File: contributing.md (hunk 3 above)
Comment (patrickbirch) [reply contributing.md:59]: leave it - we may have a developer review the technical side of the change.

File: contributing.md (hunk 5 above)
Comment (patrickbirch) [reply contributing.md:81]: Done.

File: contributing.md (hunk 6 above)
Comment (patrickbirch) [reply contributing.md:128]: Fixed.

File: contributing.md (hunk 8 above)
Comment (patrickbirch) [reply contributing.md:69]: Added.

File: contributing.md (hunk 10 above)
Comment (patrickbirch) [reply contributing.md:118]: Done.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**